.github/workflows/main.yaml

```diff
@@ -24,8 +24,10 @@ jobs:
     strategy:
       matrix:
         test_kubernetes_target: [current, prev, prev2]
+        test_scheduler_manifest: [daemonset, deployment]
```

Comment:
Do you need to run the test on all combinations?
I feel it is too much.

Reply:
We will revise them if there are any problems.